python-generate-outputs-simple.yaml

```
@@ -0,0 +1,32 @@
+id: python-generate-outputs-simple
```

Checking why this can’t be snake case
```suggestion
id: python_generate_outputs_simple
```